Example/Emission/stub_container.js

```diff
@@ -17,13 +17,29 @@ export default class StubbedContainer extends React.Component {
         getStoreData: () => {},
         primeCache: () => {}
       },
-    route: { name: 'string' }
+    route: { name: 'string' , params:{}, useMockData: true, queries: {}}
     };
   }
 
+  // TODO: Improve handling state, currently this works only when you full re-render the whole virtual view tree
+  // e.g. jumping between two stories of the same component will not trigger state changes. I gave a
+  // considerable amount of time, then noticed we _don't_ use too much state inside our components
+  // and have made the time tradeoff to say it's not worth it for now.
+
+  // If you are using state in a component, where you would normally set the default, you should check props first
+  // e.g. `this.state = this.props.state || { following: false, followersCount: props.artist.counts.follows };`
```

Comment:
I’m not following why this is, but `this.props.state` looks really counter-intuitive. Would love some clarification on the reason and ramifications first.